--- gemrb-0.6.2/gemrb/Makefile.am~	2010-08-20 19:49:10.000000000 +0200
+++ gemrb-0.6.2/gemrb/Makefile.am	2010-08-23 09:48:03.319399915 +0200
@@ -1,10 +1,9 @@
 bin_PROGRAMS = gemrb
-sysconf_DATA = GemRB.cfg.sample.in
 gemrb_SOURCES = GemRB.cpp 
 
 gemrb_LDADD =  ./core/libgemrb_core.la
 
-SUBDIRS = core plugins includes GUIScripts override docs
+SUBDIRS = core plugins includes GUIScripts override
 EXTRA_DIST = GemRB.cfg* plugins-prepare.sh CMakeLists.txt
 
 gemrb_LDFLAGS = $(all_libraries) @LIBPTHREAD@
