--- gemrb-0.6.1/gemrb/Makefile.am~	2010-06-16 18:11:26.000000000 +0200
+++ gemrb-0.6.1/gemrb/Makefile.am	2010-06-17 13:45:59.317619658 +0200
@@ -1,10 +1,9 @@
 bin_PROGRAMS = gemrb
-sysconf_DATA = GemRB.cfg.sample GemRB.cfg.subdir.sample
 gemrb_SOURCES = GemRB.cpp 
 
 gemrb_LDADD =  ./core/libgemrb_core.la
 
-SUBDIRS = core plugins includes GUIScripts override docs
+SUBDIRS = core plugins includes GUIScripts override
 EXTRA_DIST = GemRB.cfg* plugins-prepare.sh CMakeLists.txt
 
 gemrb_LDFLAGS = $(all_libraries) @LIBPTHREAD@
