--- gemrb-0.8.7/gemrb/GemRB.cfg.sample.in.orig	2021-04-23 12:11:03.909680837 +0200
+++ gemrb-0.8.7/gemrb/GemRB.cfg.sample.in	2021-04-23 12:16:48.730617624 +0200
@@ -173,7 +173,7 @@
 #PluginsPath=@DEFAULT_PLUGINS_DIR@
 
 # This is the path where GemRB will store cached files, enter the full path.
-CachePath=@DEFAULT_CACHE_DIR@
+CachePath=~/.cache/gemrb
 
 # The path where GemRB looks for non-BAM fonts (eg. TTF)
 #CustomFontPath=
