--- Makefile.am	2005-10-19 23:57:32.000000000 +0200
+++ Makefile.am-new	2006-11-03 00:28:08.000000000 +0100
@@ -21,9 +21,9 @@
 dist_man1_MANS = gemrb.1
 
 install-data-hook:
-	rm -f $(bindir)/plugins/libgemrb_core.so
-	mkdir -p $(bindir)/Cache 2>/dev/null
-	rm -rf $(bindir)/Cache/*
+	rm -f $(DESTDIR)/$(bindir)/plugins/libgemrb_core.so
+	mkdir -p $(DESTDIR)/$(bindir)/Cache 2>/dev/null
+	rm -rf $(DESTDIR)/$(bindir)/Cache/*
 
 docs:
 	doxygen
